src/core/core.helpers.js

```diff
@@ -53,6 +53,16 @@ module.exports = function(Chart) {
 		}
 		return base;
 	};
+    // Adjust element index when min is defined
+	helpers.adjustIndex = function(element, chartConfig) {
+		var min = chartConfig.options.scales.xAxes[0].ticks.min;
```

Comment:
hmm I was looking into **scatter plot** where you can define multiple xAxes with different `ticks.min`... which is the case I think you are referring to. 

as per documentation: 

| Name       | Type          | Default  |  Description|
| ------------- |:-------------:| -----:| ---------------:|
| scales.xAxes | Array |            |  The bar chart officially supports only 1 x-axis but uses an array to keep the API consistent. Use a scatter chart if you need multiple x axes.|

how about if I make this patch specific to when only one `xAxes` is defined (Given scatter plot is only time where multi `xAxes` is possible) ?